packages/flutter/lib/src/material/data_table.dart

```diff
@@ -1013,7 +1017,7 @@ class DataTable extends StatelessWidget {
         child: label,
       );
     }
-    return label;
+    return TableCell(child: label);
```

Comment:
will this cause behavior change besides the semantics role?

Reply:
i dont think it will cause any behavior change, TableCell is just a wrapper